keyboards/keebio/quefrency/info.json

```diff
@@ -169,6 +169,201 @@
               {"label":"Down", "x":15, "y":4},
               {"label":"Right", "x":16, "y":4}
           ]
+      },
```

Comment:
The changes to this file should be reverted.

Reply:
I didn't mean to submit this as part of the PR.  I believe I reverted it in e0e20af.  Is there a way to delete this (62a226d)?

Reply:
The most straight-forward way, IMO:

```
git checkout a6857d18 -- keyboards/keebio/quefrency/info.json
git commit
git push
```

That'll edit the file so it matches the current QMK master version, and update this pull request.